src/main.go

```diff
@@ -131,10 +153,41 @@ func init() {
 	}
 }
 
+func init() {
+	// init dir GANTED_LOG_DIR
+	gantedLogDir := getEnv("GANTED_LOG_DIR", "/var/log/ganted")
+	if _, err := os.Stat(gantedLogDir); os.IsNotExist(err) {
+		os.Mkdir(gantedLogDir, 0755)
```

Comment:
Shall we `panic()` if the log directory doesn't exist and can't be created either?